app/src/main/java/com/ncnf/authentication/ui/SignUpFragment.java

```diff
@@ -89,7 +84,6 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         });
     }
 
-    @RequiresApi(api = Build.VERSION_CODES.N)
     public void signUp() {
```

Comment:
Should there be input validation?

Reply:
It is done two lines below that with checkInputs()